functions/contains.yml

```diff
@@ -38,17 +38,17 @@ examples:
   args: [foobar, 123]
   returns: false
 - context: ["a", "b"]
-  args: [@, a]
+  args: ["@", a]
```

Comment:
Even though this syntax is technically correct, it feels "uncomfortable" to me. 

Do we need to have such a fine grained structure to the level of each individual argument?

If the goal is to support generating the documentation, would a simple `expression` property be all right?

Something like:

```yaml
- given: ["a", "b"]
- expression: "contains(@, `\"a\"`)"
- result: true
```
Or

```yaml
- given: ["a", "b"]
- expression: 'contains(@, `"a"`)'
- result: true
```

Or better yet:

```yaml
- given: ["a", "b"]
- expression: >-
   contains(@, `"a"`)
- result: true
```
I think if the goal is to include those as part of the compliance tests, this makes it also more consistent with the syntax of those test files.

What do you think?

Reply:
Yes, the idea is to move the compliance tests into this structure. I am planning on reworking the compliance test syntax to better utilize yaml features and deduplicate a lot of the test data.

I wanted to keep the arguments split apart so that a implementations function unit tests could call the function directly rather than have to pipe the test through the entire JMESPath evaluation. This of course makes testing expression-type arguments slightly more complicated within a unit test, but the implementer has the choice at that point for what direction they want to take for that function.

Reply:
> I do not like the inconsistency between different syntax for the the first and second argument.

I see what you are saying, while yaml permits simple unquoted strings, we could require all strings to be quoted as part of a convention.